### ISSUE TITLE
Choosing the "edit page on github" link for tutorials does not really work

### ISSUE THREAD

**Issue by bennoleslie** [seL4/docs#55]
For example:

https://docs.sel4.systems/Tutorials/capabilities.html

takes you to this page:

https://github.com/seL4/docs/edit/master/Tutorials/capabilities.md

Which just has:

```{% include tutorial.md %}```

It's not clear where the actual tutorial.md source is. 

Makefile has:

```
GIT_REPOS:=$(shell (cd _data/projects && for i in `ls`; do cat $$i | ../../tools/get_repos.py ; done) | sort -u)
REPOSITORY_LIST = sel4 sel4-tutorials capdl
REPOSITORIES = $(GIT_REPOS:%=_repos/%)
```

But there doesn't appear to be an `sel4-tutorials` repo available on github.

**Comment by lsf37** [comment]
The sel4-tutorials repo is this one I think: https://github.com/SEL4PROJ/sel4-tutorials

You're right that this needs more documentation, I had trouble finding sources of some of those includes myself.

**Comment by lsf37** [comment]
(The actual page it includes is this one: https://github.com/SEL4PROJ/sel4-tutorials/blob/master/tutorials/capabilities/capabilities.md )

**Comment by lsf37** [comment]
Hey @kent-mcleod, I remember us talking about what we should be doing here to make the include links available on the live site, and that that already worked for many of them.

Do you remember what we discussed there and what should be done here?

**Comment by lsf37** [comment]
Awesome, that was quick, thanks!

**Comment by lsf37** [comment]
This should now be fixed by #65
